pyproject.toml

```diff
@@ -50,12 +50,16 @@ Tracker = "https://github.com/yt-project/widgyts/issues"
 
 [tool.hatch.build]
 artifacts = [
+    "widgyts/labextension/*.tgz",
     "widgyts/labextension",
 ]
 
+[tool.hatch.metadata]
+allow-direct-references = true
+
 [tool.hatch.build.targets.wheel.shared-data]
 "widgyts/labextension/static" = "share/jupyter/labextensions/@yt-project/yt-widgets/static"
-"install.json" = "share/jupyter/labextensions/@yt-project/yt-widgets/install.json"
+"./install.json" = "share/jupyter/labextensions/@yt-project/yt-widgets/install.json"
```

Comment:
probably didn't need the `./` here, but the cookiecutter had it....

Reply:
I think better to keep it -- I'm always a bit leery that perhaps the working directory for builds changes, or something, so following upstream seems right.